apps-script/incident-response/Stylesheet.html

```diff
@@ -0,0 +1,87 @@
+<!--
+Copyright 2023 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!-- [START chat_incident_response_stylesheet] -->
+<style>
+  * {
+    box-sizing: border-box;
+  }
+  body {
+    font-family: Roboto, Arial, Helvetica, sans-serif;
+  }
+  div.container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: absolute;
+    top: 0; bottom: 0; left: 0; right: 0;
+  }
+  div.content {
+    width: 80%;
+    max-width: 1000px;
+    padding: 1rem;
+    border: 1px solid #999;
+    border-radius: 0.25rem;
+    box-shadow: 0 2px 2px 0 rgba(66, 66, 66, 0.08), 0 2px 4px 2px rgba(66, 66, 66, 0.16);
+  }
+  h1 {
+    text-align: center;
+    padding-bottom: 1rem;
+    margin: 0 -1rem 1rem -1rem;
+    border-bottom: 1px solid #999;
+  }
+  div#output {
```

Comment:
```suggestion
 #output {
```

id should already be unique within the document